feat: adds mvp limits page (#23)
momentohq/public-dev-docs: docs/limits.md

---
sidebar_position: 4
sidebar_class_name: sidebar-item-limits
---

# Limits 

> We have several default limits in place at Momento to help protect you as you get up and running. This page will outline our default service limits that you should be aware of as you look at using Momento as a caching provider. Note that all the limits can be adjusted up as needed. Please reach out to hello@momentohq.com to increase limits if the defaults are too low. (Self service coming soon!)

Reviewer: ```suggestion
This page outlines our default service limits at Momento. Note that limits can be adjusted as needed, please reach out to hello@momentohq.com with your requests. Self service coming soon!
```

> We have several default limits in place at Momento to help protect you as you get up and running. This page will outline our default service limits that you should be aware of as you look at using Momento as a caching provider. Please note that all the limits can be adjusted up as needed, we can work with you to adjust to support your workloads. Please reach out to hello@momentohq.com to increase limits if default are to low. (Self service coming soon!)

Reviewer 2: Copyedit:

```suggestion
We have several default limits in place at Momento to help protect you as you get up and running. This page will outline our default service limits that you should be aware of as you look at using Momento as a caching provider. Note that all the limits can be adjusted up as needed. Please reach out to hello@momentohq.com to increase limits if the defaults are too low. (Self service coming soon!)
```

| Limit             | Value         |
|-------------------|---------------|
| Item Size         | 1MB           |
| Max Cache Count   | 10            |

> | Data Plane TPS    | 100 TPS/Cache |

Reviewer: ```suggestion
| Data Plane TPS    | 100 TPS/Cache (up to 1MB/s) |
```

> | Control Plane TPS | 20 TPS        |

Reviewer 2: Define "TPS" before this

Author: ack

Reviewer 3: i thought i remembered a convo somewhere where we had decided to go with "RPS" instead of "TPS" to avoid confusion with rdbms transactions?

Author: updated!
